app/controllers/admin/bets_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::BetsController < Admin::BaseController
+  include EventsHelper
+
+  before_action :load_match, only: %i(create new)
+  before_action :check_status_match, only: :create
+
+  def new
+    flag = create_all_bet @soccer_match
+    return if flag.nil?
+
+    action_if_create_false unless flag
+
+    flash[:success] = t ".bets_default"
```

Comment:
ko cần method `create all bet ` mô

```suggestion
    return unless check_bet_empty? match

    arr = Settings.arr_content_bets
    
      SoccerMatch.transaction do
        arr.each_with_index do |content, i|
          match.bets.create!(rate: 1, bet_type: i + 1, content: content)
        end
      end
      flash[:success] = t ".bets_default"
    rescue StandardError
      action_if_create_false
    end
```

Reply:
em đã sữa lại và handle trực tiếp vào create rồi ạ!